src/Microsoft.DotNet.Build.CloudTestTasks/UploadClient.cs

```diff
@@ -135,11 +135,12 @@ public string EncodeBlockIds(int numberOfBlocks, int lengthOfId)
             }
 
             string blockListUploadUrl = blobUploadUrl + "?comp=blocklist";
-            DateTime dt1 = DateTime.UtcNow;
+
             using (HttpClient client = new HttpClient())
             {
                 Func<HttpRequestMessage> createRequest = () =>
                 {
+                    DateTime dt1 = DateTime.UtcNow;
```

Comment:
These variable names appear to be really inconsistent ("dt", "dt1", "dateTime").